docs_src/OVMS/pipelinerun.md

```diff
@@ -17,56 +18,18 @@ You can run the pipeline script, `run.sh` with a given pipeline profile via the
     - xeon
 3. [Environment Variables](../dev-tools/environment_variables.md)
  
-Run the command based on your requirement. You have to get your choices for #1-4 above to start the pipeline run, see [details](#run-pipeline-with-different-input-sourceinputsrc-types) section below.
+Run the command based on your requirement. Once choices are selected for #1-3 above, to start the pipeline run, use the commands from the [Examples](#run-pipeline-with-different-input-sourceinputsrc-types) section below.
 
+## Examples using different input source types
+In the following examples, [environment variables](../dev-tools/environment_variables.md) are used to select the desired `PIPELINE_PROFILE` and `RENDER_MODE`. This table uses `run.sh` to run the object_detection pipeline profile:
 
-### Environment variables
-When running run.sh script, we support environment variables as input for containers. [Here is a list of environment variables and how to apply them](../dev-tools/environment_variables.md)
+| Input source Type  | Input Source Parameter                |Command                                                                                                                                        |          
+|---------------------|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| Simulated camera   | `rtsp://127.0.0.1:8554/camera_0`      | <code>PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0</code>      |
```

Comment:
For the object detection pipeline it would be optimal to use camera_1 which by default is the coca cola video. camera_0 is currently the barcode which may produce odd results.